src/declarations/stencil-private.ts

```diff
@@ -1559,48 +1563,86 @@ export type ComponentRuntimeMetaCompact = [
   ComponentRuntimeHostListener[]?
 ];
 
+/**
+ * Runtime metadata for a Stencil component
+ */
 export interface ComponentRuntimeMeta {
+  /**
+   * This number is used to hold a series of bitflags for various features we
+   * support on components. The flags which this value is intended to store are
+   * documented in the `CMP_FLAGS` enum.
```

Comment:
I tried to do a link for these but for some reason it wasn't showing up in the rendered hints in my editor...I'll fiddle with it more

Reply:
turns out you just need a `module:path/to/module` directive!

Reply:
Hmmm do you have any further reading on that syntax? It doesn't seem to work with Intellisense for IntelliJ or VS Code 🤔

Reply:
hmm so based on https://github.com/microsoft/TypeScript/issues/43869 and https://github.com/microsoft/TypeScript/issues/43950 it seems that TypeScript doesn't currently support resolving symbols which aren't imported in the current module correctly

which is a bit unfortunate...it works correctly if I add `import { CMP_FLAGS } from '../utils/constants'` but then we get an unused import error, because referencing a symbol in a JSDoc doesn't mark it as used

Reply:
all of this makes me inclined to either leave it as a non-working `{@link CMP_FLAGS}` which may start to work at some point in the future or just `CMP_FLAGS` and the reader can search project symbols to find it (there's only one variable thusly named at present)

Reply:
Ahh the things I take for granted in IntelliJ 😆 I agree with you - I say we stick with `@link` or how you had it originally. Sorry for the wild goose chase 😞

Reply:
👍 no worries, I'll go with just `{@link CMP_FLAGS}` in the hope that at some point in the future it starts working